### PR TITLE
BaSP-TG-104: Projects page

### PR DESCRIPTION
**Pull request by Ansitox**
Add table with projects filtered by the employee's Id
Add 'Add hours'  button linked to the timesheets form

QA:
How to test?:

The home page must show the list of projects associated to the current employee (in this PR we used the ID of the employee called DONOT DELETE)
The  'Add hours' button must redirect you to the timesheets form

### CODE REVIEW

File: src/Components/Employees/Home/index.js [hunk 1]
```diff
@@ -1,14 +1,123 @@
...
+      for (let i = 0; i < projects.employees.length; i++) {
+        result = projects.employees[i].employeeId?._id === id;
+      }
+    }
+    return result;
+  });
+
+  const headers = [
+    'Project Name',
+    'Desription',
```
Comment (matescarabino) [inline src/Components/Employees/Home/index.js:60]: Fix typo

```suggestion
    'Description',
```

**Review by matescarabino — CHANGES_REQUESTED** [review]
Fix the typo. Everything else looks and works fine!

**Review by matescarabino — APPROVED** [review]

**Review by renzopimentel — APPROVED** [review]

**Review by CristianBucci — APPROVED** [review]
Everything looks fine!

**Review by MaxiG-dev — APPROVED** [review]
Looks good for me! ✅

**Review by melissabonillasellanes — APPROVED** [review]
Everything looks fine!

**Review by NicoSan02 — APPROVED** [review]

**Pull request merged**